test/src/main/java/org/axonframework/test/aggregate/FixtureConfiguration.java

```diff
@@ -160,16 +162,28 @@
 
     /**
      * Registers a resource that is eligible for injection in handler method (e.g. methods annotated with {@link
-     * CommandHandler @CommandHandler}, {@link
-     * EventSourcingHandler @EventSourcingHandler} and {@link
-     * EventHandler @EventHandler}. These resource must be
-     * registered <em>before</em> registering any command handler.
+     * CommandHandler @CommandHandler}, {@link EventSourcingHandler @EventSourcingHandler} and {@link EventHandler
      *
-     * @param resource The resource eligible for injection
+     * @param resource the resource eligible for injection
      * @return the current FixtureConfiguration, for fluent interfacing
+     * @EventHandler}. These resource must be registered <em>before</em> registering any command handler.
```

Comment:
Just a weird javadoc identation/new line here.

Reply:
Good catch, will fix this.